parametermanager/api/ParameterManager.Samples.Tests/ParameterManagerFixture.cs

```diff
@@ -115,6 +138,63 @@ private void DeleteParameter(ParameterName name)
         }
     }
 
+    public KeyRing GetKeyRing(string projectId, string keyRingId)
+    {
+        string name = $"projects/{projectId}/locations/global/keyRings/{keyRingId}";
+        try
+        {
+            KeyRing resp = KmsClient.GetKeyRing(name);
+            return resp;
+        }
+        catch (Grpc.Core.RpcException e) when (e.StatusCode == Grpc.Core.StatusCode.NotFound)
+        {
+            throw new Exception($"KeyRing with ID '{keyRingId}' in project '{projectId}' does not exist.");
+        }
+    }
+
+    public CryptoKey CreateHsmKey(string projectId, string keyId, string keyRingId)
+    {
+        KeyRingName parent = new KeyRingName(projectId, "global", keyRingId);
+
+        CreateCryptoKeyRequest request = new CreateCryptoKeyRequest
+        {
+            ParentAsKeyRingName = parent,
+            CryptoKeyId = keyId,
+            CryptoKey = new CryptoKey
+            {
+                Purpose = CryptoKey.Types.CryptoKeyPurpose.EncryptDecrypt,
+                VersionTemplate = new CryptoKeyVersionTemplate
+                {
+                    Algorithm = CryptoKeyVersion.Types.CryptoKeyVersionAlgorithm.GoogleSymmetricEncryption,
+                    ProtectionLevel = ProtectionLevel.Hsm,
+                },
+            },
+        };
+
+        try
+        {
+            string name = $"projects/{projectId}/locations/global/keyRings/{keyRingId}/cryptoKeys/{keyId}";
+            CryptoKey resp = KmsClient.GetCryptoKey(name);
```

Comment:
Why do we need to get the key first. Aren't we creating these as we need them and deleting them on dispose? Or is this a similar case as the key ring? If it is a similar case as the key ring let's address it in the same way we addressed the key ring case.